testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveCoercion.java

```diff
@@ -127,10 +127,14 @@ private static HiveTableDefinition.HiveTableDefinitionBuilder tableDefinitionBui
                         "    longdecimal_to_longdecimal            DECIMAL(20,12)," +
                         //"    float_to_decimal           " + floatType + "," + // this coercion is not permitted in Hive 3. TODO test this on Hive < 3.
                         //"    double_to_decimal          DOUBLE," + // this coercion is not permitted in Hive 3. TODO test this on Hive < 3.
-                        "    decimal_to_float           DECIMAL(10,5)," +
-                        "    decimal_to_double          DECIMAL(10,5)," +
-                        "    varchar_to_bigger_varchar  VARCHAR(3)," +
-                        "    varchar_to_smaller_varchar VARCHAR(3)" +
+                        "    decimal_to_float                   DECIMAL(10,5)," +
+                        "    decimal_to_double                  DECIMAL(10,5)," +
+                        "    short_decimal_to_varchar           DECIMAL(10,5)," +
+                        "    long_decimal_to_varchar            DECIMAL(20,12)," +
+                        "    short_decimal_to_bounded_varchar    DECIMAL(10,5)," +
```

Comment:
I guess you changed the surrounding lines so that this is aligned, but at least on the Github UI it doesn't look aligned.